_includes/new_footer_en.html

```diff
@@ -0,0 +1,51 @@
+<footer class="home-footer">
+  <div class="columns-container row justify-content-between">
+    <div class="column-item col-md-4 col-sm-6 col-xs-12 social">
+      <header>
+        <div class="subtitle small centered">
+          <div class="underline light">
+            <div class="text"><h4>Stay in touch</h4></div>
+          </div>
+        </div>
+      </header>
+      <div class="content">
+        <div class="twitter">
+          <a href="https://twitter.com/p2pu">
+            <span><i class="fa fa-twitter" aria-hidden="true"></i>@P2PU</span>
+          </a>
+        </div>
+        <div class="facebook">
+          <a href="https://www.facebook.com/P2PUniversity">
+            <span><i class="fa fa-facebook" aria-hidden="true"></i>P2PUniversity</span>
+          </a>
+        </div>
+        <div class="email">
+          <a href="mailto:thepeople@p2pu.com">
+            <span><i class="fa fa-envelope" aria-hidden="true"></i>thepeople@p2pu.com</span>
+          </a>
+        </div>
+      </div>
+    </div>
+
+    <div class="column-item col-md-4 col-sm-6 col-xs-12 offset-md-4 newsletter">
+      <header>
+        <div class="subtitle small centered">
+          <div class="underline light">
+            <div class="text"><h4>Get the newsletter</h4></div>
+          </div>
+        </div>
+      </header>
+      <div class="content">
+        <form action="">
+          <input type="text" placeholder="Your email address">
+          <input type="submit" class="btn p2pu-btn light" value="subscribe">
```

Comment:
We'll need to add the hidden fields and correct names to integrate with mailchimp